docker-image/Dockerfile.amd64

```diff
@@ -14,6 +14,9 @@ COPY --from=base /sbin/tini /sbin/tini
 ADD bin/calico-typha-amd64 /code/calico-typha
 ADD typha.cfg /etc/calico/typha.cfg
 
+# Add nsswitch.conf so that we correctly resolve localhost based on /etc/hosts.
```

Comment:
There won't be a `/etc/hosts` file will there?

Reply:
Pretty sure Kubernetes populates one for you: https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/#default-hosts-file-content